tests/unit/plugins/inventory/test_flightctl.py

```
@@ -530,6 +535,359 @@ def test_validate_device_partial_expression_with_missing_field(self):
+        })
+
+        config = module._setup_connection_configuration()
+
+        mock_oidc.assert_called_once_with(
+            'https://rhem.example.com', 'admin', 'redhat', True, '/etc/pki/tls/custom-ca.crt'
+        )
+        self.assertEqual(config.access_token, 'oidc-token')
+
+    @patch("plugins.inventory.flightctl.open_url")
+    def test_oidc_grant_passes_ca_path_to_open_url(self, mock_open):
+        def _resp(body):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_resp`

(ANN202)